src/Symfony/Component/OptionsResolver/OptionConfigurator.php

```diff
@@ -134,4 +134,16 @@ public function info(string $info): static
 
         return $this;
     }
+
+    /**
+     * Sets whether ignore undefined options.
+     *
+     * @return $this
+     */
+    public function ignoreUndefined(bool $ignore = true): static
+    {
+        $this->resolver->ignoreUndefined($ignore);
+
+        return $this;
```

Comment:
It should return the resolver, no?

Reply:
this is at least weird on my opinion to have optionResolver to be returned at this point.
I as a user would not expect such behaviour.
please make this decision without me and i'll update the PR